pykob/config.py

```diff
@@ -761,7 +761,11 @@ def read_config():
         __key = __WIRE_KEY
         _wire = user_config.get(__CONFIG_SECTION, __key)
         if (_wire) or (_wire.upper() != "NONE"):
-            wire = int(_wire)
+            try:
+                wire = int(_wire)
+            except ValueError as ex:
+                log.err("Wire number value '{}' is not a valid integer value.".format(_wire))
+                wire = 1
```

Comment:
Agree. Odd thing is, that I thought I had that - obviously not. Only difference is that I set the wire to 0.